app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -3,146 +3,176 @@ package org.sil.storyproducer.model
+            PhaseType.COMMUNITY_WORK -> R.drawable.ic_people_white_48dp
+            PhaseType.ACCURACY_CHECK -> R.drawable.ic_school_white_48dp
             PhaseType.WHOLE_STORY -> R.drawable.ic_school_white_48dp
             PhaseType.REMOTE_CHECK -> R.drawable.ic_school_white_48dp
-            PhaseType.BACKT -> R.drawable.ic_headset_mic_white_48dp
-            PhaseType.DRAMATIZATION -> R.drawable.ic_mic_box_48dp
+            PhaseType.BACK_T -> R.drawable.ic_headset_mic_white_48dp
+            PhaseType.VOICE_STUDIO -> R.drawable.ic_mic_box_48dp
             else -> R.drawable.ic_mic_white_48dp
         }
     }
 
+    /**
+     * get the color associated with a phase
+     * @return color
+     */
+    fun getColor() : Int {
+        return when(phaseType){
+            PhaseType.LEARN -> R.color.learn_phase
+            PhaseType.TRANSLATE_REVISE -> R.color.translate_revise_phase
+            PhaseType.COMMUNITY_WORK -> R.color.comunity_work_phase
+            PhaseType.ACCURACY_CHECK -> R.color.accuracy_check_phase
+            PhaseType.VOICE_STUDIO -> R.color.voice_studio_phase
+            PhaseType.FINALIZE -> R.color.finalize_phase
+            PhaseType.SHARE -> R.color.share_phase
+            PhaseType.BACK_T -> R.color.backT_phase
+            PhaseType.WHOLE_STORY -> R.color.whole_story_phase
+            PhaseType.REMOTE_CHECK -> R.color.remote_check_phase
+            else -> R.color.black
+        }
+    }
+
+    /**
+     * get path of audio file needed by slide, based on the current phase (narration or a draft)
+     * @return String
+     */
     fun getReferenceAudioFile(slideNum: Int = Workspace.activeSlideNum) : String {
         val filename = when (phaseType){
-            PhaseType.DRAFT -> Workspace.activeStory.slides[slideNum].narrationFile
-            PhaseType.COMMUNITY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.CONSULTANT_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.DRAMATIZATION -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.BACKT -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.TRANSLATE_REVISE -> Workspace.activeStory.slides[slideNum].narrationFile
+            PhaseType.COMMUNITY_WORK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.ACCURACY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.VOICE_STUDIO -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.BACK_T -> Workspace.activeStory.slides[slideNum].chosenDraftFile
             else -> ""
         }
         return Story.getFilename(filename)
     }
 
+    /**
+     * TODO: refactor
+     * the following 3 functions are used when displaying the phase names and creating paths to store audio files
+     * (see Issue #547 for conversation on fix)
+     */
     fun getPrettyName() : String {

```

Comment:
This provides the Phase name shown in the UI, correct?

Reply:
I know, its very confusing lol. I'll add better comments to these functions.